platforms/corda-ent/charts/nmap/templates/deployment.yaml

```diff
@@ -85,13 +82,11 @@ spec:
                     sleep {{ $.Values.healthcheck.readinesscheckinterval }}
                   else
                    initial_param=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["network-parameters-initial"]')
-                   echo "${initial_param}" | base64 -d > ${MOUNT_PATH}/notary-nodeinfo/network-parameters-initial.conf
+                   echo "${initial_param}" | base64 -d > ${NODEINFO_MOUNT_PATH}/network-parameters-initial.conf
                    echo "Successfully got initial network parameters"
                    break
                 fi
                 COUNTER=`expr "$COUNTER" + 1`
-              done
```

Comment:
Please add 'done' back again. Otherwise chart wont run

Reply:
Fixed in commit [7568f6c](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/844/commits/7568f6cb9b32e8f50b7d860d97e999f40da288eb)